.codespell-whitelist

```diff
@@ -1 +1,2 @@
 mut
+crate
```

Comment:
Also not sure why, it's a normal English word?

Reply:
I don't know. The codespell suggests crate -> create.